### PR TITLE
[stable/pgadmin] new helm chart for pgAdmin

### PR DESCRIPTION
**Pull request by rowanruseler**
#### What this PR does / why we need it:
Was missing a decent pgadmin helm chart

#### Special notes for your reviewer:
Created a simple pgadmin helm chart, colleague suggested perhaps get it added to incubator. Apologies if something is incorrect.

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Review by cpanato — CHANGES_REQUESTED** [review]
please move to `stable` folder

**Comment by rowanruseler** [comment]
> please move to `stable` folder

Done, and updated the appVersion cause it was the default one rather then the version of pgadmin.

**Comment by rowanruseler** [comment]
@cpanato Is there perhaps something missing, or should something be different?

File: stable/pgadmin/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: pgAdmin is a web based administration tool for PostgreSQL database
+name: pgadmin
+version: 0.1.1
```
Comment (cpanato) [inline stable/pgadmin/Chart.yaml]: should be 
```suggestion
version: 1.0.0
```
since it is in the stable folder and will be the first release

**Review by cpanato — CHANGES_REQUESTED** [review]

**Comment by rowanruseler** [comment]
@cpanato sorry about those commits, need to get used to the signoff.

**Comment by cpanato** [comment]
@rowanruseler no problem you can rebase if you want

**Review by cpanato — APPROVED** [review]
/approve
/lgtm

**Pull request merged**